components/HelloWorld.vue

```diff
@@ -1,7 +1,185 @@
 <template>
-  <div>Hello world!</div>
+  <div>
+
+    <header>
+      <nav>
+        <div id="logo-container">
+          <a tag="img" href="/">
+            <img src="~/public/bitcamp-brand/logos/logotype.png" id="logo-image" />
+          </a>
+        </div>
+        <ul class=" nav-pages">
+          <li class="page">
+            <a href="/our-mission">Our Mission</a>
+          </li>
+          <li class="page">
+            <a href="/tracks">Tracks</a>
+          </li>
+          <li class="dropdown-page">
+            <a href="/mini-events" class="dropdown-text">Mini Events</a>
+            <div class="dropdown-elements-container">
+              <div class="dropdown-elements">
+                <ul>
+                  <li>
+                    <a href="/mini-events/event1">Value</a>
+                  </li>
+                  <li>
+                    <a href="/mini-events/event2">Value</a>
+                  </li>
+                  <li>
+                    <a href="/mini-events/event3">Value</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="page">
+            <a href="/campfire-games">Campfire Games</a>
+          </li>
+          <li class="page">
+            <a href="/schedule">Schedule</a>
+          </li>
+          <li class="page">
+            <a href="/faq">FAQ</a>
+          </li>
+          <li class="page">
+            <a href="/sponsors">Sponsors</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="divider">
+    </div>
+
+    <div>
+      <slot />
+    </div>
+  </div>
 </template>
 
-<script setup lang="ts"></script>
+<style scoped lang="scss">
+$bitcamp: #FF6F3F;
```

Comment:
Thanks for the feedback! 

I ran into an issue with one part.

When hovering on the dropdown tab's elements, the background is the bitcamp color with reduced opacity, to achieve this, I used background-color: rgba($bitcamp, 0.3). 

It worked since my bitcamp variable was referencing the hex code before. However, with it referencing var(--color-bitcamp), the variable is being taken literally by scss and not being compiled as the hex code. I haven't found a solution to this yet so I substituted the bitcamp hex code in for now. Is there a way to remedy this?

Reply:
That's a great question, if these were scss variables we could just use `rgba($bitcamp, 0.3)`.

Here's a hacky solution that'll work for now. We can use the `::before` pseudo-element for the background and apply the opacity there, so that the text isn't affected.
```
.dropdown-elements li:hover a {
  position: relative;
  color: $bitcamp;
  width: 100%;
  text-decoration: none !important;

  &::before {
    content: '';
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    background-color: $bitcamp;
    opacity: 0.3;
    border-radius: 0.5rem;
    z-index: -1;
  }
}
```